parts/k8s/kubernetescustomscript.sh

```diff
@@ -257,6 +257,11 @@ function installCNI() {
     tar -xzf $CONTAINERNETWORKING_CNI_TGZ_TMP -C $CNI_BIN_DIR
     chown -R root:root $CNI_BIN_DIR
     chmod -R 755 $CNI_BIN_DIR
+    # Turn on br_netfilter (needed for the iptables rules to work on bridges)
+    # and permanently enable it
+    modprobe br_netfilter
```

Comment:
Let's do `retrycmd_if_failure 10 5 10 modprobe br_netfilter` above to appease the runtime provisioning gods.